charts/streams-app-cleanup-job/templates/job.yaml

```diff
@@ -37,8 +37,10 @@ spec:
           env:
             - name: ENV_PREFIX
               value: {{ .Values.configurationEnvPrefix }}_
-            - name: "{{ .Values.configurationEnvPrefix }}_STREAMS_CONFIG"
-              value: "{{- range $key, $value := .Values.streams.config }}{{ $key }}={{ $value }},{{- end }}"
+            {{- range $key, $value := .Values.streams.config }}
+            - name: {{ printf "STREAMS_%s" $key | replace "." "_" | upper | quote }}
+              value: {{ $value | quote }}
+            {{- end }}
```

Comment:
> This already possible, isn't it? You can set stream config parameters via env variables starting with `ENVPREFIX_STREAM_CONFIG_`.

With that env variable you can only set the whole streams config at once. We want to set some (especially `group.instance.id`) from a Kubernetes field without affecting the others

Reply:
I see. I thought it would be possible to set individual fields with the current parser. Makes sense then!

Reply:
> I think it would be helpful to add an example in the values.yaml

There is already an example :-) The helm chart itself did not change, just the Kubernetes Resources are slightly different